pkgs/desktops/expidus/file-manager/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, flutter, fetchFromGitHub }:
+flutter.buildFlutterApplication rec {
+  pname = "expidus-file-manager";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "ExpidusOS";
+    repo = "file-manager";
+    rev = version;
+    sha256 = "sha256-aAPmwzNPgu08Ov9NyRW5bcj3jQzG9rpWwrABRyK2Weg=";
+  };
+
+  depsListFile = ./deps.json;
+  vendorHash = "sha256-mPGrpMUguM9XAYWH8lBQuytxZ3J0gS2XOMPkKyFMLbc=";
+
+  postInstall = ''
+    rm $out/bin/file_manager
+    ln -s $out/app/file_manager $out/bin/expidus-file-manager
+
+    mkdir -p $out/share/applications
+    mv $out/app/data/com.expidusos.file_manager.desktop $out/share/applications
+
+    mkdir -p $out/share/icons
+    mv $out/app/data/com.expidusos.file_manager.png $out/share/icons
+
+    mkdir -p $out/share/metainfo
+    mv $out/app/data/com.expidusos.file_manager.metainfo.xml $out/share/metainfo
+
+    substituteInPlace "$out/share/applications/com.expidusos.file_manager.desktop" \
+      --replace "Exec=file_manager" "Exec=$out/bin/expidus-file-manager" \
+      --replace "Icon=com.expidusos.file_manager" "Icon=$out/share/icons/com.expidusos.file_manager.png"
+  '';
+
+  disallowedReferences = [
+    flutter
+    flutter.unwrapped
```

Comment:
```
error: output '/nix/store/d98grmgli8psici4ll1j5zy96rrr80rp-expidus-file-manager-0.1.2-debug' is not allowed to refer to the following paths:
         /nix/store/jkx6ckjhybkmiq80nlbakv5hln38s4dn-flutter-3.10.5-unwrapped
```

Reply:
Huh, didn't get that locally. I pushed a fix.